docs/introduction/flocker-plugin.rst

```diff
@@ -64,3 +64,10 @@ This is the "ultimate integration demo", a pre-built demo environment that inclu
 
 Flocker also has planned integrations with major orchestration tools such as Docker Swarm, Kubernetes and Apache Mesos.
 More information on these integrations is :ref:`available in the Labs section <labs-projects>`.
+
+Get Started with the Flocker Plugin for Docker
+==============================================
+
+Before you can install the plugin, you'll need to complete the :ref:`full-installation` instructions.
```

Comment:
The Full Installation includes:
* The CLI which I don't think is necessary, is it?
* The node services which are necessary
* The Flocker plugin for Docker, which makes this a circular dependency

If I'm right, replace this with something that links to the node install docs.